hibernate-core/src/test/java/org/hibernate/orm/test/mapping/embeddable/strategy/usertype/embedded/Person.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or http://www.gnu.org/licenses/lgpl-2.1.html
+ */
+package org.hibernate.orm.test.mapping.embeddable.strategy.usertype.embedded;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.hibernate.annotations.CompositeType;
+
+import jakarta.persistence.AttributeOverride;
+import jakarta.persistence.Column;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+
+@Table(name = "people")
+//tag::embeddable-usertype-property[]
+@Entity
+public class Person {
+	@Id
+	public Integer id;
+	@Embedded
+	@AttributeOverride(name = "firstName", column = @Column(name = "first_name"))
+	@AttributeOverride(name = "lastName", column = @Column(name = "last_name"))
+	@CompositeType( NameCompositeUserType.class )
+	public Name name;
+	@ElementCollection
+	@AttributeOverride(name = "firstName", column = @Column(name = "first_name"))
+	@AttributeOverride(name = "lastName", column = @Column(name = "last_name"))
+	@CompositeType( NameCompositeUserType.class )
+	public Set<Name> aliases;
```

Comment:
would be great to insert empty lines between fields to improve code readability for the code would be in the doc and it deserves perfection